src/librustdoc/html/static/js/scrape-examples.js

```diff
@@ -60,8 +60,8 @@
 
         const locs = example.locs;
         if (locs.length > 1) {
-            const next = createScrapeButton(buttonHolder, "next", "≻");
-            const prev = createScrapeButton(buttonHolder, "prev", "≺");
+            const next = createScrapeButton(buttonHolder, "next", ">");
+            const prev = createScrapeButton(buttonHolder, "prev", "<");
```

Comment:
In the past, we've tried to move away from using punctuation as icons, because it looks confusingly like syntax. We could use the same unicode icons as the `expand` buttons, or we could use SVG. https://github.com/rust-lang/rust/pull/136950

```suggestion
            const next = createScrapeButton(buttonHolder, "next", "→");
            const prev = createScrapeButton(buttonHolder, "prev", "←");
```